bagit.xml

```diff
@@ -1146,8 +1146,8 @@ the <spanx style="emph">checksum</spanx> and, under Windows, the backslashes ins
           <t>
 Implementers &may; wish to accept this format by ignoring a leading asterisk or
 handling differences in line termination gracefully but, if so, implementations
-&must; warn the user that the bag in question will fail strict validation. In
-such cases it is strongly encouraged that tools provide an easy option to
+&should; warn the user that the bag in question will fail strict validation. In
```

Comment:
The MUST is only if dealing with a `*`, and they choose to ignore it and try and process the bag. Given that, I am ok with saying it should still be MUST since it is the choice of the implementor to not be strict about the Bagit spec.

Reply:
My thought was also that in this case it's not asking for any behavioural change other than a compatibility warning.